### PR TITLE
Bug 1751147: Correct ca-bundle filename in terminate on change

### PR DESCRIPTION
**Pull request by mfojtik**

### CODE REVIEW

File: glide.lock [hunk 1]
```diff
@@ -1,8 +1,8 @@
 hash: 22a791d3e967426e426b9bd64362616ab63455865112b6c2e00dd3d712cab593
-updated: 2019-08-29T11:01:12.504797+02:00
```
Comment (sttts) [inline glide.lock]: `bump(*): library-go: cmd: run terminate-on-files observer #524` would give a nice history one can read

**Comment by mfojtik** [comment]
Added glide bump to pickup an observer bug in library-go, testing locally.

File: glide.lock (hunk 1 above)
Comment (mfojtik) [reply glide.lock]: done

**Comment by stlaz** [comment]
/hold
You still need something like https://github.com/openshift/cluster-authentication-operator/pull/194/files#diff-c37e62a9d7e53c40a57f44b1b9780494R534 to fix the BZ

**Comment by mfojtik** [comment]
@stlaz fixed in the last commit, PTAL

**Comment by mfojtik** [comment]
The file observer reaction to `/var/run/configmaps/trusted-ca-bundle/ca-bundle.crt` change was tested manually and the container successfully restart.

File: pkg/operator2/operator.go [hunk 2]
```diff
@@ -521,7 +530,12 @@ func (c *authOperator) checkDeploymentReady(deployment *appsv1.Deployment, opera
 func (c *authOperator) checkRouteHealthy(route *routev1.Route, routerSecret *corev1.Secret, ingress *configv1.Ingress) (ready bool, msg, reason string, err error) {
 	caData := routerSecretToCA(route, routerSecret, ingress)
 
-	rt, err := transportFor("", caData, nil, nil)
+	// if systemCABundle is not empty, append the new line to the caData
+	if len(c.systemCABundle) > 0 {
+		caData = append(bytes.TrimSpace(caData), []byte("\n")...)
```
Comment (stlaz) [inline pkg/operator2/operator.go]: nit: no need to trim the spaces here, there can be multiple empty lines between the certs
https://goplay.space/#2tCqSuOBWJL

**Comment by stlaz** [comment]
/lgtm
based on an offline discussion, I'm fine with the newline

**Pull request merged**